File: F.csm301/python/main.py
import sys

def read_file(filepath):
    try:
        with open(filepath, "r", encoding="utf-8") as f:
            return f.read()
    except Exception as e:
        return "Error: " + str(e)

if __name__ == "__main__":
    if len(sys.argv) > 1:
        path = sys.argv[1]
    else:
        path = "text.txt"
    content = read_file(path)
    print(content)

File: F.csm301/Main.java
import java.io.BufferedReader;
import java.io.FileReader;
import java.io.IOException;

public class Main {

    public static void main(String[] args) {
        String filepath = "C:\\Users\\User\\OneDrive - Mongolian University of Science and Technology\\Documents\\F.csm301\\text.txt";
        String content = readFile(filepath);
        System.out.println(content);
    }

    public static String readFile(String filepath) {
        StringBuilder sb = new StringBuilder();

        try (BufferedReader reader = new BufferedReader(new FileReader(filepath))) {
            String line;
            while ((line = reader.readLine()) != null) {
                sb.append(line).append("\n");
            }
        } catch (IOException e) {
            return "Error: " + e.getMessage();
        }

        return sb.toString().trim();
    }
}
